src/app/ubs/ubs-admin/components/shared/components/modal-text/modal-text.component.html

```diff
@@ -6,14 +6,17 @@
     <p class="modal-text">{{ text | translate }}{{ text2 | translate: { bagName: bagName } }}</p>
   </div>
   <div class="d-flex justify-content-end buttons" mat-dialog-actions>
-    <button class="secondary-global-button btn m-0 mr-2" mat-button (click)="onNoClick()">
+    <button *ngIf="modalData.name != 'cancel'" class="secondary-global-button btn m-0 mr-2" mat-button (click)="onNoClick(false)">
```

Comment:
you can create method  check(val) { return  val === 'cancel'}  and used it in *ngIf  directives. It prevent code duplications))

Reply:
Method is created. Thanks!